pkg/controller/release/release.go

```diff
@@ -202,6 +205,14 @@ func (c *connector) Connect(ctx context.Context, mg resource.Managed) (managed.E
 		}
 	}
 
+	if proxy := p.Spec.Proxy; proxy != "" {
```

Comment:
Wouldn't it work by passing proxy/no_proxy environment variables via a ControllerConfig ? https://www.oreilly.com/library/view/security-with-go/9781788627917/5ea6a02b-3d96-44b1-ad3c-6ab60fcbbe4f.xhtml

Reply:
This would work when we only need to connect to a single proxy instance per controller.

In our case we use one crossplane/helm provider instance to manage multiple clusters in different vpc networks. Therefore we need to connect to a different proxy instance depending of the cluster/vpc.

Reply:
@turkenh

Reply:
Hi @turkenh ,

Is there any update on this PR?